doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -166,11 +166,18 @@ nRF Machine Learning (Edge Impulse)
 nRF Desktop
 -----------
 
+* Added:
+
+  * :ref:`nrf_desktop_swift_pair_app`.
+    The module is used to enable or disable the Swift Pair Bluetooth advertising payload depending on the selected Bluetooth peer (used local identity).
+
```

Comment:
```suggestion
* Added the :ref:`nrf_desktop_swift_pair_app`. The module is used to enable or disable the Swift Pair Bluetooth advertising payload depending on the selected Bluetooth peer (used local identity).

```
Merged since it's one item for now.